terraform/services/rds-api/main.tf

```
@@ -0,0 +1,10 @@
+resource "aws_db_instance" "rds" {
```

The pertinent info here is that this is a database instance for the API

```suggestion
resource "aws_db_instance" "api" {
```